<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>T4addline_1d_wizard</class>
 <widget class="QWizard" name="T4addline_1d_wizard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wizard for adding a special incrementor</string>
  </property>
  <property name="wizardStyle">
   <enum>QWizard::ModernStyle</enum>
  </property>
  <property name="titleFormat">
   <enum>Qt::RichText</enum>
  </property>
  <property name="subTitleFormat">
   <enum>Qt::RichText</enum>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <property name="title">
    <string>The name of this incrementor contains the word &quot;&lt;font color=&quot;#ff0000&quot;&gt;&lt;b&gt;WHEN&lt;/b&gt;&lt;/font&gt;&quot;. </string>
   </property>
   <property name="subTitle">
    <string/>
   </property>
   <attribute name="pageId">
    <string notr="true"/>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="textLabel3">
      <property name="text">
       <string>&quot;&lt;font color=&quot;#ff0000&quot;&gt;&lt;b&gt;WHEN&lt;/b&gt;&lt;/font&gt;&quot; - means that for some events this incrementor may not have the meaningful 
value.&lt;/font&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="textLabel3_2">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;It may happen when the detector didn't fire in particular event, &lt;br/&gt;or the data was not good enough (lower than some threshold),&lt;br/&gt;or the calculation was not possible.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="indent">
       <number>35</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="_2">
      <item>
       <spacer name="spacer17">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>201</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_for_example_when">
        <property name="text">
         <string>for example...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="spacer18">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>46</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="textLabel2_incr_name">
      <property name="text">
       <string>Incrementor name</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 246, 246);
border-color: rgb(89, 38, 37);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>What to do if (in the particular event) the incrementor is not currently valid ?</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="textLabel1_2">
           <property name="text">
            <string>Let this line (of the condition) be treated as:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QRadioButton" name="radioButton_obligatory">
             <property name="text">
              <string>false (recommended)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_if_possible">
             <property name="text">
              <string>true</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="textLabel1_3_2">
        <property name="text">
         <string>&lt;font size=&quot;-1&quot;&gt;&lt;font color=&quot;#005500&quot;&gt;&lt;i&gt;Here you usually want the option: &quot;false&quot; &lt;br&gt;(This means that we are interested only in the events where this incrementor has the meaningful value).&lt;/i&gt;&lt;/font&gt;&lt;/font&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer20">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>31</width>
          <height>71</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage2">
   <property name="title">
    <string>&lt;font size=&quot;+0&quot;&gt;The name of this incrementor contains the words:  &quot;&lt;font color=&quot;#0000ff&quot;&gt;&lt;b&gt;ALL&lt;/b&gt;&lt;/font&gt;&quot; and  &quot;&lt;font color=&quot;#ff0000&quot;&gt;&lt;b&gt;WHEN&lt;/b&gt;&lt;/font&gt;&quot;</string>
   </property>
   <attribute name="pageId">
    <string notr="true"/>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="textLabel3_3">
      <property name="text">
       <string>&quot;&lt;font color=&quot;#0000ff&quot;&gt;&lt;b&gt;- ALL&lt;/b&gt;&lt;/font&gt;&quot; - means that acutally this is a set of incrementors.
&lt;br&gt;
&quot;&lt;font color=&quot;#ff0000&quot;&gt;&lt;b&gt;- WHEN&lt;/b&gt;&lt;/font&gt;&quot; - means that some of the items (members of the &quot;ALL set&quot;) 
may not be present in every event.&lt;/font&gt;
(The single item is  present only if its &quot;WHEN&quot; situation in true).</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="_4">
      <item>
       <spacer name="spacer14">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>461</width>
          <height>21</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_expl_all_when">
        <property name="text">
         <string>for example...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="spacer15">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>21</width>
        <height>59</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="textLabel2_incr_name_2">
      <property name="text">
       <string>Incrementor name</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;What to do if (in the particular event) one of items of the &amp;quot;ALL&amp;quot; set -&lt;br/&gt;is currently NOT valid ?&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textFormat">
          <enum>Qt::RichText</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="textLabel1_2_2">
           <property name="text">
            <string>Let this item  be considered  (in the condition) as:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QRadioButton" name="radioButton_always_ALL">
             <property name="text">
              <string>false</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="radioButton_if_possible_ALL">
             <property name="text">
              <string>true (recommended)</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="textLabel1_3">
        <property name="text">
         <string>&lt;font size=&quot;-1&quot;&gt;&lt;font color=&quot;#005500&quot;&gt;&lt;i&gt;For the keyword &quot;ALL&quot; - you usually want the option: &quot;true&quot;.&lt;br&gt;
This means that we are interested in events when at least one the incrementors had a valid data&lt;/i&gt;&lt;/font&gt;&lt;/font&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignVCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="indent">
         <number>7</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer16">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>21</width>
          <height>31</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="wizardPage_what_was_chosen">
   <property name="title">
    <string>Your decision was...</string>
   </property>
   <attribute name="pageId">
    <string notr="true"/>
   </attribute>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="spacer2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>21</width>
          <height>61</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="textLabel1">
        <property name="text">
         <string>You have chosen the incrementor:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom</set>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="textLabel_incrementor_name">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>&lt;h3&gt;Name of your incrementor&lt;/h3&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="textLabel2">
        <property name="text">
         <string>In the condition table it will be marked as:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="textLabel2_2">
          <property name="text">
           <string>&lt;h3&gt;When incrementor is not valid?  :&lt;/h3&gt;</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel_when_to_test">
          <property name="text">
           <string>&lt;h3&gt;always&lt;/h3&gt;</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="spacer4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Preferred</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>41</width>
          <height>38</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
